### PR TITLE
make requirements/*.txt output to stdout

### PR DESCRIPTION
**Pull request by ewjoachim**
Also, fixes a doc inconsistency on the subject

### CODE REVIEW

File: docs/development/patterns.rst [hunk 1]
```diff
@@ -81,7 +81,7 @@ process for adding new dependencies:
 1. Remove the project name from the appropriate ``.in`` file
 2. From the repositories root directory, recompile the dependencies for each modified ``.in`` file::
```
Comment (di) [inline docs/development/patterns.rst:82]: I think this line needs updated too

Comment (ewjoachim) [reply docs/development/patterns.rst:82]: Hm ? I'm not sure I understand how it should be replaced?

The best I understand from this is that you imply we should talk about the `.txt` file ? Like:
```suggestion
2. From the repositories root directory, recompile the ```.txt` for each modified ``.in`` file::
```

Comment (di) [reply docs/development/patterns.rst:82]: Sorry, read this too fast, you're right

**Comment by ewjoachim** [comment]
(Thank you ❤️ )

**Pull request merged**